src/components/LeftPanel.tsx

```diff
@@ -67,39 +86,107 @@ const LeftPanel = ({ currentTheme }: ConnectWalletProps) => {
         </ListItem>
       </List>
       <Divider style={{ backgroundColor: colors[currentTheme].accent2 }} />
-      <List style={{ backgroundColor: colors[currentTheme].accent2 }}>
+      <List
+        style={{ backgroundColor: colors[currentTheme].accent2 }}
+        component="nav"
+        aria-labelledby="nested-list-subheader"
+      >
         <div>
           <ListItem>
-            <ListItemButton component={Link} to={'/mining-pool'}>
+            <ListItemButton component={Link} to={'/dashboard'}>
               <ListItemIcon>
-                <Hardware style={{ color: colors[currentTheme].secondary }} />
+                <Home style={{ color: colors[currentTheme].secondary }} />
               </ListItemIcon>
-              <ListItemText style={{ color: colors[currentTheme].secondary }} primary="Mining Pool" />
-            </ListItemButton>
-          </ListItem>
-          <Divider variant="middle" style={{ backgroundColor: colors[currentTheme].secondary }} />
-        </div>
-        <div>
-          <ListItem>
-            <ListItemButton component={Link} to={'/voting'}>
-              <ListItemIcon>
-                <Poll style={{ color: colors[currentTheme].secondary }} />
-              </ListItemIcon>
-              <ListItemText style={{ color: colors[currentTheme].secondary }} primary="Voting" />
-            </ListItemButton>
-          </ListItem>
-          <Divider variant="middle" style={{ backgroundColor: colors[currentTheme].secondary }} />
-        </div>
-        <div>
-          <ListItem>
-            <ListItemButton component={Link} to={'/cinema'}>
-              <ListItemIcon>
-                <Movie style={{ color: colors[currentTheme].secondary }} />
-              </ListItemIcon>
-              <ListItemText style={{ color: colors[currentTheme].secondary }} primary="Cinema" />
+              <ListItemText style={{ color: colors[currentTheme].secondary }} primary="Dashboard" />
             </ListItemButton>
           </ListItem>
         </div>
+        {currentRole !== 'Viewer' && (
```

Comment:
Was this `===`? Or is it correct? What happens for the other roles?

Reply:
The default role is Viewer. You are supposed to have access only to the dashboard if you are not logged in. Access to the profile you have when you are from NormalUser above.